src/Generating/ProtIntGen.h

```diff
@@ -47,7 +47,7 @@ class cProtIntGen
 	virtual ~cProtIntGen() {}
 
 	/** Generates the array of specified size into a_Values, based on given min coords. */
-	virtual void GetInts(int a_MinX, int a_MinZ, int a_SizeX, int a_SizeZ, int * a_Values) = 0;
+	virtual void GetInts(size_t a_MinX, size_t a_MinZ, size_t a_SizeX, size_t a_SizeZ, int * a_Values) = 0;
```

Comment:
This change is all wrong. The `a_MinX` and `a_MinZ` are world coordinates and are allowed to be negative. The size, although it should never be negative, should be a compatible type because of common math operations combining the two.